scanners/zap/zap.py

```diff
@@ -191,6 +191,9 @@ def _setup_zap_cli(self):
         """
         self.zap_cli.extend(self._get_standard_options())
 
+        # Addon update has already been done, if enabled. Prevent a new check for update
+        self.zap_cli.append("-silent")
```

Comment:
Just wondering if this option will place nice with the `-addonupdate` option, ie if a user has `updateAddons=true` in config, will the `-silent` option prevent this from working, or maybe cause the command to fail?

Reply:
It indeed does not play nice with `-addonupdate`, but this is why I added the option in `_setup_zap_cli()` and not in `_get_standard_options()` : unless I missed something in the call tree, the `-silent` should be added only on the main ZAP command, not the command that deals with installing and updating plugins.
Unless I missed something, it should be fine (it looked fine in my tests)